diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/generation/Generation.kt

```diff
@@ -58,6 +59,8 @@ private fun generateWarningNames() {
 
 private fun validateYear() {
     val file = File("diktat-rules/src/test/resources/test/paragraph2/header/CopyrightDifferentYearExpected.kt")
+    log.info("===================================")
```

Comment:
What about other files? We could check an entire directory `paragraph2/header`, I think.